.travis.yml

```diff
@@ -28,17 +28,16 @@ matrix:
     # development dependencies
     - php: 7.4
       env: DEPENDENCIES='dev'
-    - php: nightly
-      env: COMPOSER_FLAGS="--ignore-platform-reqs" SYMFONY_DEPRECATIONS_HELPER="max[self]=0"
+    - php: 8.0
```

Comment:
It seems `8.0snapshot` is in use in api-platform, I suppose we could do the same